src/azure/Model/CodeModelTSa.cs

```diff
@@ -116,8 +116,11 @@ public override bool HasMappers()
         public override string GenerateMapperIndex()
         {
             TSBuilder builder = new TSBuilder();
-            builder.Comment(AutoRest.Core.Settings.Instance.Header);
-            builder.Line();
+
+            if (AutoRest.Core.Settings.Instance.Header != "") {
```

Comment:
You could also use
```cs
if (!string.IsNullOrWhitespace(AutoRest.Core.Settings.Instance.Header))
```

It would check for null, empty and a string consisting only in white spaces ([docs](https://docs.microsoft.com/en-us/dotnet/api/system.string.isnullorwhitespace?view=netcore-3.1))